rllib/models/torch/modules/multi_head_attention.py

```diff
@@ -5,12 +5,14 @@
 """
 from ray.rllib.utils.framework import try_import_torch
 from ray.rllib.models.torch.misc import SlimFC
+from ray.rllib.utils.annotations import OldAPIStack
 from ray.rllib.utils.torch_utils import sequence_mask
 from ray.rllib.utils.framework import TensorType
 
 torch, nn = try_import_torch()
 
 
+@OldAPIStack
```

Comment:
What's actually the state on attention? WIll the new stack get encoders and heads with attention? Also, I see more often in papers an MLP encoder followed by an LSTM head - maybe we should provide an LSTM head in the same breath, too.